### DISCUSSION TITLE
Clarification on GitHub Education Benefits Eligibility

### DISCUSSION THREAD

**Discussion by hukai916** [community/community#146858] [GitHub Education]
### Hi everyone,

Dear GitHub Education Community,

I recently submitted an application for GitHub Education benefits, as I am a bioinformatics researcher at a recognized medical school. According to the GitHub Education policy ("Be an **educator, faculty member, or researcher** at a recognized educational institution."), I understand that **researchers** at accredited institutions may qualify for these benefits.

However, after submitting my application along with a document that verifies my position as a bioinformatician, I received a rejection email stating: "Your document does not appear to indicate you as a **faculty** member."

I would appreciate clarification on whether the benefits are available exclusively to faculty members, or if researchers—such as myself—are also eligible under the current policy.

Thank you for your time and assistance.

Best regards,

**Comment by stickyburn** [comment]
They're not gonna do shit. It seems like the program is only for checking a box that they care about education. Look at how terrible their UX is, it says everything.
